tests/integration_tests/instances.py

```diff
@@ -70,18 +71,41 @@ def __init__(
         self._ip = ""
 
     def destroy(self):
+        wait = True
         if isinstance(self.instance, GceInstance):
-            self.instance.delete(wait=False)
+            wait = False
+        for i in range(1, 32):
```

Comment:
Think we should have some kind of wait timer here? Otherwise we might be spending 15 minutes per test waiting for a delete that will never succeed?

Reply:
@TheRealFalcon thoughts?

Reply:
Sounds like a good idea to me

Reply:
https://github.com/canonical/cloud-init/pull/5926